.github/workflows/fuzz.yml

```diff
@@ -0,0 +1,47 @@
+name: Fuzz
+on:
+  schedule:
+    # Once a week
+    - cron: '0 0 * * 0'
+  push:
+    paths:
+    - '.github/workflows/fuzz.yml'
+  # Allow manual trigger
+  workflow_dispatch:
+
+env:
+  CARGO_INCREMENTAL: 0
+  CARGO_NET_RETRY: 10
+  CI: 1
+  RUST_BACKTRACE: short
+  RUSTFLAGS: "-D warnings -W unreachable-pub -W bare-trait-objects"
+  RUSTUP_MAX_RETRIES: 10
+
+jobs:
+  rust:
+    if: github.repository == 'rust-lang/rust-analyzer'
```

Comment:
Let's drop this `if`.

Reply:
Actually, let's not drop it :smile:.

Reply:
I'm just going to fiddle with the triggers a little, to see if I can the manual workflow_dispatch triggers working properly. At the moment I can trigger the workflow manually be the job get's cancelled because of this if statement. 

https://github.com/silvergasp/rust-analyzer/actions

Reply:
Manual triggers work, there's one in the `release` workflow, which I've used a couple of times.

Reply:
Yeah the manual trigger is working fine. I just wanted to test the workflow manually before this get's merged but because my fork != rust-lang/rust-analyzer when I manually trigger it, the job get's cancelled because the if statement fails.